LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/parsing/font-palette-values-invalid-expected.txt

```diff
@@ -1,6 +1,6 @@
 
 PASS CSS Fonts Module Level 4: parsing @font-palette-values
-PASS CSS Fonts Module Level 4: parsing @font-palette-values 1
```

Comment:
We have another issue here (which was hidden by our bug of bug) https://bugs.webkit.org/show_bug.cgi?id=260066